docs/upgrade.html

```diff
@@ -45,10 +45,8 @@ <h5><a id="upgrade_400_notable" href="#upgrade_400_notable">Notable changes in 4
             <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-896%3A+Remove+old+client+protocol+API+versions+in+Kafka+4.0">KIP-896</a> for the details.
         </li>
         <li>
-            Before updating your Kafka cluster which build on Zookeeper to Kafka 4.0, you must understand what the updates and changes 
-            are between Zookeeper mode and Kraft mode. See the following page to learn about some of the significant changes in 
-            latest Kafka releases.
-            <!--#include virtual="zk2kraft.html" -->
+            Before updating your Kafka cluster built on Zookeeper to Kafka 4.0, you must understand the updates and changes
+            between Zookeeper mode and Kraft mode. See <a href="/{{version}}/zk2kraft">Removed Zookeeper Configurations and Metrics</a> in latest Kafka releases for more information.
```

Comment:
@frankvicky Just a tiny suggestion to improve clarity:

```suggestion
            between Zookeeper mode and Kraft mode. See <a href="/{{version}}/zk2kraft">Removing Zookeeper Configurations and Metrics</a> in latest Kafka releases for more information.
```